src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -10,7 +10,7 @@ export default class GuideDrawer extends React.Component {
     guide: PropTypes.object.isRequired,
     step: PropTypes.number.isRequired,
     onClose: PropTypes.func.isRequired,
```

Comment:
One last change here, can we rename this to onFinish? Otherwise it's unclear what the diff. between onDismiss and onClose is.